### ISSUE TITLE
NPE on ITI-62 transaction in IPF 3.2.0

### ISSUE THREAD

**Issue by Thopap** [oehf/ipf#150]
I have updated from IPF 3.2-rc2 to IPF 3.2.0: Now the following NPE occure during a ITI-62 transaction:

`java.lang.NullPointerException
	at org.openehealth.ipf.commons.ihe.xds.core.ebxml.ebxml30.EbXMLRemoveObjectsRequest30.getId(EbXMLRemoveObjectsRequest30.java:126) ~[ipf-commons-ihe-xds-3.2.0.jar:?]
	at org.openehealth.ipf.commons.ihe.xds.core.validate.requests.RemoveObjectsRequestValidator.validate(RemoveObjectsRequestValidator.java:44) ~[ipf-commons-ihe-xds-3.2.0.jar:?]
	at org.openehealth.ipf.platform.camel.ihe.xds.XdsCamelValidators.lambda$20(XdsCamelValidators.java:255) ~[ipf-platform-camel-ihe-xds-3.2.0.jar:?]
`

From the change history it looks like #140 have introduced this issue.

Basically we build the XDS ITI-62 request with the following code and send it to the ITI-62 endpoint:
`    private RemoveObjectsRequest requestWithDeleteReference(String... uuids) {
        RemoveObjectsRequest request = new RemoveObjectsRequest();
        request.setObjectRefList(new ObjectRefListType());
        for(String uuid : uuids) {
            ObjectRefType reference = new ObjectRefType();
            reference.setId(uuid);
            request.getObjectRefList().getObjectRef().add(reference);
        }
        return request;
    }`
